internal/clients/telegram/response/response.go

```diff
@@ -37,18 +110,100 @@ func (m SendMessage) SetParseMode(mode option.Option[string]) SendMessage {
 	return m
 }
 
+// EnableWebPreview enables the preview that is visible below the message and displays the webpage content.
+func (m SendMessage) EnableWebPreview() SendMessage {
+	m.DisableWebpagePreview = true
+
+	return m
+}
+
+// EnableWebPreview enables the preview that is visible below the message and displays the webpage content.
+func (m SendMessage) DisableWebPreview() SendMessage {
+	m.DisableWebpagePreview = false
+
+	return m
+}
+
 func (m SendMessage) URLEncode() (string, url.Values) {
 	var (
 		endpoint = "sendMessage"
 		params   = url.Values{}
 	)
 
-	params.Add("chat_id", string(m.ChatID))
-	params.Add("text", m.Text)
+	params.Set("chat_id", string(m.ChatID))
+	params.Set("text", m.Text)
 
 	if parseMode, isSome := m.ParseMode.Unwrap(); isSome {
-		params.Add("parse_mode", parseMode)
+		params.Set("parse_mode", parseMode)
+	}
+
+	if !m.DisableWebpagePreview {
```

Comment:
Confusing statement (also check the bools that had to be inverted in the method)